index.html

```diff
@@ -0,0 +1,214 @@
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li id="cc_campaign"><a href="#">RoR Campaign</a></li>
+        </ul>
+      </div>
+    </section>
+
+    <section id="hero">
+      <header id="hero-header">
+        <h3>"Hello World!"</h3>
+        <h1>
+          Aruba Ruby <br>
+          Global summit <br>
+          2023
+        </h1>
+      </header>
+      <article>
+        <p id="sub-theme">
+          Aruba Ruby Global Summit is an annual gathering for all who are
+          passionate about Ruby and Ruby on Rails. The Aruba based conference has served
+          the Ruby community since 2008.
+        </p>
+        <div id="date-venue">
+          <h3 id="date">2023.01.04(THU) ~ 05(FRI)</h3>
+          <h5 id="venue">@Cas di Cultura, Oranjestad, Aruba</h5>
+        </div>
+      </article>
+    </section>
+
+    <section id="program">
+      <header>
+        <h1>Main Program</h1>
+        <div id="bar"></div>
+      </header>
+      <article>
+        <ul>
+          <li class="program-card">
+            <img src="./assets/images/program_icon_01.png" alt="lecture" />
+            <h3>Lecture</h3>
+            <p>
+              Listen to speakers from various countries about how they've used
+              Ruby on Rails to build applications.
+            </p>
+          </li>
+          <li class="program-card">
+            <img src="./assets/images/program_icon_02.png" alt="exhibition" />
+            <h3>RoR Exhibition</h3>
+            <p>
+              Appreciate various creations using Ruby on Rails, organized by
+              Microverse
+            </p>
+          </li>
+          <li class="program-card">
+            <img src="./assets/images/program_icon_03.png" alt="forum" />
+            <h3>Forum</h3>
+            <p>
+              Have the time to share your thoughts and opinions with topic
+              experts.
+            </p>
+          </li>
+          <li class="program-card">
+            <img src="./assets/images/program_icon_04.png" alt="workshop" />
+            <h3>Workshop</h3>
+            <p>
+              Create new applications leveragin the power of open source code,
+              frameworks, and libraries.
+            </p>
+          </li>
+          <li class="program-card">
+            <img src="./assets/images/program_icon_05.png" alt="affiliates" />
+            <h3>RoR ignite</h3>
+            <p>
+              Get opportunities to network with Ruby on Rails affiliates around
+              the world, also after the summit.
+            </p>
+          </li>
+        </ul>
+      </article>
+      <article id="see_program">
+        <a href="#">See the whole the program</a>
+      </article>
+      <button type="button" class="mobile-only">
+        Join Aruba Ruby on Rails Global Summit 2023
+      </button>
+    </section>
+
+    <section id="speakers">
+      <header>
+        <h1>Featured Speakers</h1>
```

Comment:
- [ ] It is html best practice to have only one main title in a webpage which you already have one up. Kindly use any of the remaining heading `hx (2, 3, 4, 5, 6).`